testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -31,51 +34,53 @@
 import com.datastax.oss.driver.api.core.cql.Row;
 import com.datastax.oss.driver.api.core.cql.SimpleStatement;
 import com.datastax.oss.driver.api.core.data.TupleValue;
+import com.datastax.oss.driver.api.core.data.UdtValue;
 import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
 import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
 import com.datastax.oss.driver.api.core.servererrors.InvalidQueryException;
 import com.datastax.oss.driver.api.core.type.DataTypes;
 import com.datastax.oss.driver.api.core.type.TupleType;
+import com.datastax.oss.driver.api.core.type.UserDefinedType;
 import com.datastax.oss.driver.internal.core.loadbalancing.DcInferringLoadBalancingPolicy;
 import net.jcip.annotations.NotThreadSafe;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
 
 @RunWith(Parameterized.class)
 @NotThreadSafe
 public class CQLTest extends BaseOsgiIntegrationTest
 {
-    private static final Logger logger = LoggerFactory.getLogger(PersistenceTest.class);
-
     @Rule
     public TestName name = new TestName();
 
     private String table;
     private String keyspace;
-    private CqlSession session;
+    private static CqlSession session;
```

Comment:
Nit: this module pulls the Java driver as a transitive dependency of `persistence-api`. Since it uses the driver directly, I think it would be better to declare it in its own POM.

Reply:
Nice catch!